sunpy/net/dataretriever/sources/swap.py

```diff
@@ -0,0 +1,107 @@
+''' SWAPClient definition '''
+
+__author__ = "Sudarshan Konge"
+__email__ = "sudk1896@gmail.com"
+
+import datetime
+import urllib2
+
+
+from astropy import units as u
+from bs4 import BeautifulSoup
+from sunpy.net.dataretriever.client import GenericClient
+from sunpy.util.scraper import Scraper
+
+from sunpy.time import TimeRange
+
+__all__ = ['SWAPClient']
+
+class SWAPClient(GenericClient):
+    
+    """ Returns a list of URLS to Proba2 SWAP files corresponding to value of input timerange.
+    URL source: http://proba2.oma.be/swap/data/bsd/
+
+    The earliest date available is from 24-Nov-2009
+
+    Parameters
+    ----------
+    timerange: sunpy.time.TimeRange
+        time range for which data is to be downloaded.
+        Example value - TimeRange('2015-12-30 00:00:00','2015-12-31 00:01:00')
+    
+    Instrument: Fixed argument = 'swap'
+
+    Returns
+    -------
+    urls: list
+        list of urls corresponding to requested time range.
+
+    Examples
+    --------
+    >>> from sunpy.time.timerange import TimeRange
+    >>> from sunpy.net.vso.attrs import Time,Instrument,Source,Level
+    >>> from sunpy.net.dataretriever.client import QueryResponse
+    >>> from sunpy.net.dataretriever.sources import swap
+
+    >>> LCClient = swap.SWAPClient()
+
+    >>> qr = LCClient.query(Time('2015-12-30 00:00:00','2015-12-31 00:05:00'),Instrument('swap'))
+    >>> res = LCClient.get(qr)
+    >>> dl = res.wait()
```

Comment:
To get a doctest that we can compare we would need something to see, for example, print the `len(qr)` or the url or  name of some file.